utils/SnapPy/Snappy/EEMEP/PostProcess.py

```diff
@@ -56,46 +107,88 @@ def convert_files(self, instantFilename, averageFilename):
         os.rename(averageFilename, newAvgFile)
 
 
-class Accumulate:
-    def __init__(self, path, logger) -> None:
-        self.path = path
 
-        if logger:
-            self.logger = logger
-        else:
-            self.logger = StderrLogger()
 
-    def accumulate_file(self, filename) -> None:
-        self.logger._write_log(f"Accumulating outputs found in {self.path}")
+    def accumulate_and_toa_nuc_files(self, instantFilename, averageFilename) -> None:
+        self.logger._write_log(f"Accumulating nuclear outputs found in {self.path}")
+        # rename files, make them available to further processes
+        timestamp = self.timestamp.strftime("%Y%m%dT%H%M%S")
+        simulationstart = self.timestamp.strftime("%Y-%m-%d_%H:%M:%S")
 
         dir = Path(self.path)
 
-        new_filename = filename.split(".")[0] + "_acc.nc"
-
-        nc_path = dir / filename
-        new_path = dir / new_filename
-
-        shutil.copy(nc_path, new_path)
-
-        with Dataset(new_path, 'a') as nc:
-            for var in list(nc.variables):
-                if var not in ['lon', 'lat', 'lev', 'P0', 'hyam', 'hybm', 'ilev', 'hyai', 'hybi', 'time', 'PS']:
-
-                    self.logger._write_log(f"Accumulating {var}")
-                    acc = nc.createVariable(f'{var}_ACC','f4',('time','lat','lon'), zlib=True)
-
-                    acc_data = np.cumsum(nc[var][:], axis=0)
-                    unit = nc[var].units
-                    unit_new = unit.split("/")[0] + " h/" + unit.split("/")[1]
-
-                    acc.units = unit_new
-                    acc.coordinates = nc[var].coordinates
-
-                    acc[:] = acc_data
-
-            nc.sync()
+        new_filename = dir / 'eemep_nuc_{}.nc'.format(timestamp)
```

Comment:
Use an f-string here

Reply:
equivalent, and used in this way throughout the complete file